### ISSUE TITLE
generateAccessorMethods in GVUserDefaults.m generate unnecessary accessor methods

### ISSUE THREAD

**Issue by NazgulLee** [gangverk/GVUserDefaults#29]
`generateAccessorMethods` in GVUserDefaults.m not only generate accessor methods for properties in  user custom categories, but also generate accessor methods for properties in GVUserDefaults.m itself, namely `mapping` and `userDefaults`. Probably this is unnecessary?

**Comment by kevinrenskers** [comment]
You're probably right :) 
Would you like to submit a PR to fix this?

**Comment by NazgulLee** [comment]
Yes, I'd love to have a try. And sorry for replying so late, cause I'm not very familiar with Github yet :)
